src/load_inputs/load_inputs.jl

```diff
@@ -76,7 +80,12 @@ function is_period_map_necessary(inputs::Dict)
 	multiple_rep_periods = inputs["REP_PERIOD"] > 1
 	has_stor_lds = !isempty(inputs["STOR_LONG_DURATION"])
 	has_hydro_lds = !isempty(inputs["STOR_HYDRO_LONG_DURATION"])
-    multiple_rep_periods && (has_stor_lds || has_hydro_lds)
+	if !isempty(inputs["VRE_STOR"])
+		has_vre_stor_lds = !isempty(inputs["VS_LDS"])
+	else
+		has_vre_stor_lds = false
+	end
+    multiple_rep_periods && (has_stor_lds || has_hydro_lds || has_vre_stor_lds)
```

Comment:
These can be united via `&&` since if the first evaluates `false` the second won't be tried.